src/AspNetCore/Src/applicationinfo.cpp

```diff
@@ -127,7 +127,7 @@ APPLICATION_INFO::EnsureApplicationCreated()
     // load assembly and create the application
     if (m_pConfiguration->QueryHostingModel() == APP_HOSTING_MODEL::HOSTING_IN_PROCESS)
     {
-        hr = LoadAssemblyFromLocalBin();
+        hr = LoadAssemblyFromInetsrv();
```

Comment:
Let's have the in process scenario load from inetsrv (or IISExpress) until we have the logic to look in the runtimestore/bin.